docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -166,17 +162,30 @@ data:
 EOF
 ----
 +
+[NOTE]
 The secret `nova-cell<X>-compute-config` is auto-generated for each
-`cell<X>`. You must specify `nova-cell<X>-compute-config` and `nova-migration-ssh-key` for each custom `OpenStackDataPlaneService` related to the Compute service.
+`cell<X>`. You must specify `nova-cell<X>-compute-config` and `nova-migration-ssh-key` for each custom `OpenStackDataPlaneService` CR that is related to the Compute service.
```

Comment:
Maybe:
"You must specify the values for the `nova-cell<X>-compute-config` and `nova-migration-ssh-key` parameters for each custom `OpenStackDataPlaneService` CR that is related to the Compute service."

Reply:
being largely reworked altogether (for multi-cell) https://github.com/openstack-k8s-operators/data-plane-adoption/pull/517